### ISSUE TITLE
maDLC2.2rc2: index error during re-training

### ISSUE THREAD

**Issue by deepbraincut** [DeepLabCut/DeepLabCut#1302]
Thanks for opening this issue, and thanks for using DeepLabCut (we hope you are enjoying it ;). Please fill out the template completely, including the full "traceback" and input code that you ran to hit this error.

**Your Operating system and DeepLabCut version**

Hi DLC team,

I am using windows10, DLC-GPU, DLC Anaconda, 2.2rc2.

I've created training set and refined tracklets using DLC2.2rc1. and upgraded to DLC2.2rc2 and tried to expend the training set. 
I've corrected config files for identity==true for the new training set (for the first data set I did not changed it to true), and had to changed cropping size to create the new training set. I did not change any other options in GUI.
After I changed cropping size, I could create training set and checked there is a new folder iteration-1, but when I tried to start new training, there is an index error but still continues until showing up "starting multi-animal training.." I waited for hours but training did not started. I tried to delete the new folder and started from the creating a new set, did not work and had the same error. 
Could you check what is the problems? 
Please complete the following information about your system:

<details><summary>Code output</summary><p>

```python
(DLC-GPU) C:\Users\sahay>python -m deeplabcut
C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\site-packages\tensorflow\python\framework\dtypes.py:526: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\site-packages\tensorflow\python\framework\dtypes.py:527: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\site-packages\tensorflow\python\framework\dtypes.py:528: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\site-packages\tensorflow\python\framework\dtypes.py:529: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\site-packages\tensorflow\python\framework\dtypes.py:530: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\site-packages\tensorflow\python\framework\dtypes.py:535: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
Starting GUI...
currently DLCRNet is only supported in multi-animal mode
You can now check the labels, using 'check_labels' before proceeding. Then,  you can use the function 'create_training_dataset' to create the training dataset.
Found new frames..
Checking labels if they are outside the image
A training dataset file is already found for this video. The refined machine labels are merged to this data!
Merged data sets and updated refinement iteration to 1.
Now you can create a new training set for the expanded annotated images (use create_training_dataset).
 19%|███████████████▋                                                                 | 57/295 [01:47<07:29,  1.89s/it]
Traceback (most recent call last):
  File "C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\site-packages\deeplabcut\gui\create_training_dataset.py", line 376, in create_training_dataset
    self.config, n_crops, (height, width), userfeedback
  File "C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\site-packages\deeplabcut\generate_training_dataset\trainingsetmanipulation.py", line 350, in cropimagesandlabels
    raise ValueError("Crop dimensions are larger than image size")
ValueError: Crop dimensions are larger than image size
D:\DLC\MashBax_sampling_2mice-ac-2021-05-19\labeled-data\Trial1_2020-11-18-092624-0000_cropped  already exists!
100%|████████████████████████████████████████████████████████████████████████████████| 238/238 [01:40<00:00,  2.36it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 68/68 [00:28<00:00,  2.42it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:09<00:00,  2.11it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:08<00:00,  2.29it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:09<00:00,  2.13it/s]
Utilizing the following graph: [[0, 1], [0, 2], [0, 3], [0, 4], [0, 5], [0, 6], [1, 2], [1, 3], [1, 4], [1, 5], [1, 6], [2, 3], [2, 4], [2, 5], [2, 6], [3, 4], [3, 5], [3, 6], [4, 5], [4, 6], [5, 6]]
Creating training data for: Shuffle: 1 TrainFraction:  0.95
100%|████████████████████████████████████████████████████████████████████████████| 3470/3470 [00:00<00:00, 5042.44it/s]
The training dataset is successfully created. Use the function 'train_network' to start training. Happy training!
Selecting multi-animal trainer
Config:
{'all_joints': [[0], [1], [2], [3], [4], [5], [6]],
 'all_joints_names': ['Nose',
                      'LeftEar',
                      'RightEar',
                      'Center',
                      'LeftLeg',
                      'RightLeg',
                      'Tail'],
 'alpha_r': 0.02,
 'batch_size': 8,
 'clahe': True,
 'claheratio': 0.1,
 'crop_pad': 0,
 'cropratio': 0.4,
 'dataset': 'training-datasets\\iteration-1\\UnaugmentedDataSet_MashBax_sampling_2miceMay19\\MashBax_sampling_2mice_ac95shuffle1.pickle',
 'dataset_type': 'multi-animal-imgaug',
 'decay_steps': 30000,
 'deterministic': False,
 'display_iters': 500,
 'edge': False,
 'emboss': {'alpha': [0.0, 1.0], 'embossratio': 0.1, 'strength': [0.5, 1.5]},
 'fg_fraction': 0.25,
 'global_scale': 0.8,
 'histeq': True,
 'histeqratio': 0.1,
 'init_weights': 'C:\\Users\\sahay\\Anaconda3\\envs\\DLC-GPU\\lib\\site-packages\\deeplabcut\\pose_estimation_tensorflow\\models\\pretrained\\resnet_v1_50.ckpt',
 'intermediate_supervision': False,
 'intermediate_supervision_layer': 12,
 'location_refinement': True,
 'locref_huber_loss': True,
 'locref_loss_weight': 0.05,
 'locref_stdev': 7.2801,
 'log_dir': 'log',
 'lr_init': 0.0005,
 'max_input_size': 1500,
 'mean_pixel': [123.68, 116.779, 103.939],
 'metadataset': 'training-datasets\\iteration-1\\UnaugmentedDataSet_MashBax_sampling_2miceMay19\\Documentation_data-MashBax_sampling_2mice_95shuffle1.pickle',
 'min_input_size': 64,
 'mirror': False,
 'multi_stage': False,
 'multi_step': [[0.0001, 7500], [5e-05, 12000], [1e-05, 200000]],
 'net_type': 'resnet_50',
 'num_idchannel': 2,
 'num_joints': 7,
 'num_limbs': 21,
 'optimizer': 'adam',
 'pafwidth': 20,
 'pairwise_huber_loss': False,
 'pairwise_loss_weight': 0.1,
 'pairwise_predict': False,
 'partaffinityfield_graph': [[0, 1],
                             [0, 2],
                             [0, 3],
                             [0, 4],
                             [0, 5],
                             [0, 6],
                             [1, 2],
                             [1, 3],
                             [1, 4],
                             [1, 5],
                             [1, 6],
                             [2, 3],
                             [2, 4],
                             [2, 5],
                             [2, 6],
                             [3, 4],
                             [3, 5],
                             [3, 6],
                             [4, 5],
                             [4, 6],
                             [5, 6]],
 'partaffinityfield_predict': True,
 'pos_dist_thresh': 17,
 'project_path': 'D:\\DLC\\MashBax_sampling_2mice-ac-2021-05-19',
 'regularize': False,
 'rotation': 25,
 'rotratio': 0.4,
 'save_iters': 10000,
 'scale_jitter_lo': 0.5,
 'scale_jitter_up': 1.25,
 'scoremap_dir': 'test',
 'sharpen': False,
 'sharpenratio': 0.3,
 'shuffle': True,
 'snapshot_prefix': 'D:\\DLC\\MashBax_sampling_2mice-ac-2021-05-19\\dlc-models\\iteration-1\\MashBax_sampling_2miceMay19-trainset95shuffle1\\train\\snapshot',
 'stride': 8.0,
 'weigh_negatives': False,
 'weigh_only_present_joints': False,
 'weigh_part_predictions': False,
 'weight_decay': 0.0001}
Activating limb prediction...
Starting with multi-animal imgaug + adam pose-dataset loader.
Batch Size is 8
Getting specs: multi-animal-imgaug Number of Limbs: 21 No. of Joints: 7
Initializing ResNet
Loading ImageNet-pretrained resnet_50
2021-06-02 16:45:47.409689: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX AVX2
2021-06-02 16:45:47.512329: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1433] Found device 0 with properties:
name: GeForce RTX 2060 SUPER major: 7 minor: 5 memoryClockRate(GHz): 1.65
pciBusID: 0000:01:00.0
totalMemory: 8.00GiB freeMemory: 7.02GiB
2021-06-02 16:45:47.512626: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1512] Adding visible gpu devices: 0
2021-06-02 16:45:49.122163: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] Device interconnect StreamExecutor with strength 1 edge matrix:
2021-06-02 16:45:49.122216: I tensorflow/core/common_runtime/gpu/gpu_device.cc:990]      0
2021-06-02 16:45:49.122425: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1003] 0:   N
2021-06-02 16:45:49.124804: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 6736 MB memory) -> physical GPU (device: 0, name: GeForce RTX 2060 SUPER, pci bus id: 0000:01:00.0, compute capability: 7.5)
Exception in thread Thread-3:
Traceback (most recent call last):
  File "C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\threading.py", line 926, in _bootstrap_inner
    self.run()
  File "C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\site-packages\deeplabcut\pose_estimation_tensorflow\train_multianimal.py", line 76, in load_and_enqueue
    batch_np = dataset.next_batch()
  File "C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\site-packages\deeplabcut\pose_estimation_tensorflow\dataset\pose_multianimal_imgaug.py", line 279, in next_batch
    joint_ids, batch_joints, data_items, sm_size, image_shape
  File "C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\site-packages\deeplabcut\pose_estimation_tensorflow\dataset\pose_multianimal_imgaug.py", line 222, in get_targetmaps_update
    joint_ids[i], [joints[i]], data_items[i], sm_size, scale
  File "C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\site-packages\deeplabcut\pose_estimation_tensorflow\dataset\pose_multianimal_imgaug.py", line 393, in compute_target_part_scoremap_numpy
    joint_ids = joint_id[person_id].copy()
IndexError: list index out of range

Max_iters overwritten as 200000
Display_iters overwritten as 1000
Save_iters overwritten as 50000
Training parameters:
{'stride': 8.0, 'weigh_part_predictions': False, 'weigh_negatives': False, 'fg_fraction': 0.25, 'mean_pixel': [123.68, 116.779, 103.939], 'shuffle': True, 'snapshot_prefix': 'D:\\DLC\\MashBax_sampling_2mice-ac-2021-05-19\\dlc-models\\iteration-1\\MashBax_sampling_2miceMay19-trainset95shuffle1\\train\\snapshot', 'log_dir': 'log', 'global_scale': 0.8, 'location_refinement': True, 'locref_stdev': 7.2801, 'locref_loss_weight': 0.05, 'locref_huber_loss': True, 'optimizer': 'adam', 'intermediate_supervision': False, 'intermediate_supervision_layer': 12, 'regularize': False, 'weight_decay': 0.0001, 'crop_pad': 0, 'scoremap_dir': 'test', 'batch_size': 8, 'dataset_type': 'multi-animal-imgaug', 'deterministic': False, 'mirror': False, 'pairwise_huber_loss': False, 'weigh_only_present_joints': False, 'partaffinityfield_predict': True, 'pairwise_predict': True, 'all_joints': [[0], [1], [2], [3], [4], [5], [6]], 'all_joints_names': ['Nose', 'LeftEar', 'RightEar', 'Center', 'LeftLeg', 'RightLeg', 'Tail'], 'alpha_r': 0.02, 'clahe': True, 'claheratio': 0.1, 'cropratio': 0.4, 'dataset': 'training-datasets\\iteration-1\\UnaugmentedDataSet_MashBax_sampling_2miceMay19\\MashBax_sampling_2mice_ac95shuffle1.pickle', 'decay_steps': 30000, 'display_iters': 500, 'edge': False, 'emboss': {'alpha': [0.0, 1.0], 'embossratio': 0.1, 'strength': [0.5, 1.5]}, 'histeq': True, 'histeqratio': 0.1, 'init_weights': 'C:\\Users\\sahay\\Anaconda3\\envs\\DLC-GPU\\lib\\site-packages\\deeplabcut\\pose_estimation_tensorflow\\models\\pretrained\\resnet_v1_50.ckpt', 'lr_init': 0.0005, 'max_input_size': 1500, 'metadataset': 'training-datasets\\iteration-1\\UnaugmentedDataSet_MashBax_sampling_2miceMay19\\Documentation_data-MashBax_sampling_2mice_95shuffle1.pickle', 'min_input_size': 64, 'multi_stage': False, 'multi_step': [[0.0001, 7500], [5e-05, 12000], [1e-05, 200000]], 'net_type': 'resnet_50', 'num_idchannel': 2, 'num_joints': 7, 'num_limbs': 21, 'pafwidth': 20, 'pairwise_loss_weight': 0.1, 'partaffinityfield_graph': [[0, 1], [0, 2], [0, 3], [0, 4], [0, 5], [0, 6], [1, 2], [1, 3], [1, 4], [1, 5], [1, 6], [2, 3], [2, 4], [2, 5], [2, 6], [3, 4], [3, 5], [3, 6], [4, 5], [4, 6], [5, 6]], 'pos_dist_thresh': 17, 'project_path': 'D:\\DLC\\MashBax_sampling_2mice-ac-2021-05-19', 'rotation': 25, 'rotratio': 0.4, 'save_iters': 10000, 'scale_jitter_lo': 0.5, 'scale_jitter_up': 1.25, 'sharpen': False, 'sharpenratio': 0.3}
Starting multi-animal training....

(DLC-GPU) C:\Users\sahay>python -m deeplabcut
C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\site-packages\tensorflow\python\framework\dtypes.py:526: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\site-packages\tensorflow\python\framework\dtypes.py:527: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\site-packages\tensorflow\python\framework\dtypes.py:528: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\site-packages\tensorflow\python\framework\dtypes.py:529: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\site-packages\tensorflow\python\framework\dtypes.py:530: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\site-packages\tensorflow\python\framework\dtypes.py:535: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
Starting GUI...
currently DLCRNet is only supported in multi-animal mode
Selecting multi-animal trainer
Config:
{'all_joints': [[0], [1], [2], [3], [4], [5], [6]],
 'all_joints_names': ['Nose',
                      'LeftEar',
                      'RightEar',
                      'Center',
                      'LeftLeg',
                      'RightLeg',
                      'Tail'],
 'alpha_r': 0.02,
 'batch_size': 8,
 'clahe': True,
 'claheratio': 0.1,
 'crop_pad': 0,
 'cropratio': 0.4,
 'dataset': 'training-datasets\\iteration-1\\UnaugmentedDataSet_MashBax_sampling_2miceMay19\\MashBax_sampling_2mice_ac95shuffle1.pickle',
 'dataset_type': 'multi-animal-imgaug',
 'decay_steps': 30000,
 'deterministic': False,
 'display_iters': 500,
 'edge': False,
 'emboss': {'alpha': [0.0, 1.0], 'embossratio': 0.1, 'strength': [0.5, 1.5]},
 'fg_fraction': 0.25,
 'global_scale': 0.8,
 'histeq': True,
 'histeqratio': 0.1,
 'init_weights': 'C:\\Users\\sahay\\Anaconda3\\envs\\DLC-GPU\\lib\\site-packages\\deeplabcut\\pose_estimation_tensorflow\\models\\pretrained\\resnet_v1_50.ckpt',
 'intermediate_supervision': False,
 'intermediate_supervision_layer': 12,
 'location_refinement': True,
 'locref_huber_loss': True,
 'locref_loss_weight': 0.05,
 'locref_stdev': 7.2801,
 'log_dir': 'log',
 'lr_init': 0.0005,
 'max_input_size': 1500,
 'mean_pixel': [123.68, 116.779, 103.939],
 'metadataset': 'training-datasets\\iteration-1\\UnaugmentedDataSet_MashBax_sampling_2miceMay19\\Documentation_data-MashBax_sampling_2mice_95shuffle1.pickle',
 'min_input_size': 64,
 'mirror': False,
 'multi_stage': False,
 'multi_step': [[0.0001, 7500], [5e-05, 12000], [1e-05, 200000]],
 'net_type': 'resnet_50',
 'num_idchannel': 2,
 'num_joints': 7,
 'num_limbs': 21,
 'optimizer': 'adam',
 'pafwidth': 20,
 'pairwise_huber_loss': False,
 'pairwise_loss_weight': 0.1,
 'pairwise_predict': False,
 'partaffinityfield_graph': [[0, 1],
                             [0, 2],
                             [0, 3],
                             [0, 4],
                             [0, 5],
                             [0, 6],
                             [1, 2],
                             [1, 3],
                             [1, 4],
                             [1, 5],
                             [1, 6],
                             [2, 3],
                             [2, 4],
                             [2, 5],
                             [2, 6],
                             [3, 4],
                             [3, 5],
                             [3, 6],
                             [4, 5],
                             [4, 6],
                             [5, 6]],
 'partaffinityfield_predict': True,
 'pos_dist_thresh': 17,
 'project_path': 'D:\\DLC\\MashBax_sampling_2mice-ac-2021-05-19',
 'regularize': False,
 'rotation': 25,
 'rotratio': 0.4,
 'save_iters': 10000,
 'scale_jitter_lo': 0.5,
 'scale_jitter_up': 1.25,
 'scoremap_dir': 'test',
 'sharpen': False,
 'sharpenratio': 0.3,
 'shuffle': True,
 'snapshot_prefix': 'D:\\DLC\\MashBax_sampling_2mice-ac-2021-05-19\\dlc-models\\iteration-1\\MashBax_sampling_2miceMay19-trainset95shuffle1\\train\\snapshot',
 'stride': 8.0,
 'weigh_negatives': False,
 'weigh_only_present_joints': False,
 'weigh_part_predictions': False,
 'weight_decay': 0.0001}
Activating limb prediction...
Starting with multi-animal imgaug + adam pose-dataset loader.
Batch Size is 8
Getting specs: multi-animal-imgaug Number of Limbs: 21 No. of Joints: 7
Initializing ResNet
Loading ImageNet-pretrained resnet_50
2021-06-03 11:22:16.818135: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX AVX2
2021-06-03 11:22:16.896120: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1433] Found device 0 with properties:
name: GeForce RTX 2060 SUPER major: 7 minor: 5 memoryClockRate(GHz): 1.65
pciBusID: 0000:01:00.0
totalMemory: 8.00GiB freeMemory: 7.02GiB
2021-06-03 11:22:16.896279: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1512] Adding visible gpu devices: 0
2021-06-03 11:22:17.137498: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] Device interconnect StreamExecutor with strength 1 edge matrix:
2021-06-03 11:22:17.137566: I tensorflow/core/common_runtime/gpu/gpu_device.cc:990]      0
2021-06-03 11:22:17.138089: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1003] 0:   N
2021-06-03 11:22:17.138330: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 6736 MB memory) -> physical GPU (device: 0, name: GeForce RTX 2060 SUPER, pci bus id: 0000:01:00.0, compute capability: 7.5)
Exception in thread Thread-1:
Traceback (most recent call last):
  File "C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\threading.py", line 926, in _bootstrap_inner
    self.run()
  File "C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\site-packages\deeplabcut\pose_estimation_tensorflow\train_multianimal.py", line 76, in load_and_enqueue
    batch_np = dataset.next_batch()
  File "C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\site-packages\deeplabcut\pose_estimation_tensorflow\dataset\pose_multianimal_imgaug.py", line 279, in next_batch
    joint_ids, batch_joints, data_items, sm_size, image_shape
  File "C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\site-packages\deeplabcut\pose_estimation_tensorflow\dataset\pose_multianimal_imgaug.py", line 222, in get_targetmaps_update
    joint_ids[i], [joints[i]], data_items[i], sm_size, scale
  File "C:\Users\sahay\Anaconda3\envs\DLC-GPU\lib\site-packages\deeplabcut\pose_estimation_tensorflow\dataset\pose_multianimal_imgaug.py", line 393, in compute_target_part_scoremap_numpy
    joint_ids = joint_id[person_id].copy()
IndexError: list index out of range

Max_iters overwritten as 200000
Display_iters overwritten as 1000
Save_iters overwritten as 50000
Training parameters:
{'stride': 8.0, 'weigh_part_predictions': False, 'weigh_negatives': False, 'fg_fraction': 0.25, 'mean_pixel': [123.68, 116.779, 103.939], 'shuffle': True, 'snapshot_prefix': 'D:\\DLC\\MashBax_sampling_2mice-ac-2021-05-19\\dlc-models\\iteration-1\\MashBax_sampling_2miceMay19-trainset95shuffle1\\train\\snapshot', 'log_dir': 'log', 'global_scale': 0.8, 'location_refinement': True, 'locref_stdev': 7.2801, 'locref_loss_weight': 0.05, 'locref_huber_loss': True, 'optimizer': 'adam', 'intermediate_supervision': False, 'intermediate_supervision_layer': 12, 'regularize': False, 'weight_decay': 0.0001, 'crop_pad': 0, 'scoremap_dir': 'test', 'batch_size': 8, 'dataset_type': 'multi-animal-imgaug', 'deterministic': False, 'mirror': False, 'pairwise_huber_loss': False, 'weigh_only_present_joints': False, 'partaffinityfield_predict': True, 'pairwise_predict': True, 'all_joints': [[0], [1], [2], [3], [4], [5], [6]], 'all_joints_names': ['Nose', 'LeftEar', 'RightEar', 'Center', 'LeftLeg', 'RightLeg', 'Tail'], 'alpha_r': 0.02, 'clahe': True, 'claheratio': 0.1, 'cropratio': 0.4, 'dataset': 'training-datasets\\iteration-1\\UnaugmentedDataSet_MashBax_sampling_2miceMay19\\MashBax_sampling_2mice_ac95shuffle1.pickle', 'decay_steps': 30000, 'display_iters': 500, 'edge': False, 'emboss': {'alpha': [0.0, 1.0], 'embossratio': 0.1, 'strength': [0.5, 1.5]}, 'histeq': True, 'histeqratio': 0.1, 'init_weights': 'C:\\Users\\sahay\\Anaconda3\\envs\\DLC-GPU\\lib\\site-packages\\deeplabcut\\pose_estimation_tensorflow\\models\\pretrained\\resnet_v1_50.ckpt', 'lr_init': 0.0005, 'max_input_size': 1500, 'metadataset': 'training-datasets\\iteration-1\\UnaugmentedDataSet_MashBax_sampling_2miceMay19\\Documentation_data-MashBax_sampling_2mice_95shuffle1.pickle', 'min_input_size': 64, 'multi_stage': False, 'multi_step': [[0.0001, 7500], [5e-05, 12000], [1e-05, 200000]], 'net_type': 'resnet_50', 'num_idchannel': 2, 'num_joints': 7, 'num_limbs': 21, 'pafwidth': 20, 'pairwise_loss_weight': 0.1, 'partaffinityfield_graph': [[0, 1], [0, 2], [0, 3], [0, 4], [0, 5], [0, 6], [1, 2], [1, 3], [1, 4], [1, 5], [1, 6], [2, 3], [2, 4], [2, 5], [2, 6], [3, 4], [3, 5], [3, 6], [4, 5], [4, 6], [5, 6]], 'pos_dist_thresh': 17, 'project_path': 'D:\\DLC\\MashBax_sampling_2mice-ac-2021-05-19', 'rotation': 25, 'rotratio': 0.4, 'save_iters': 10000, 'scale_jitter_lo': 0.5, 'scale_jitter_up': 1.25, 'sharpen': False, 'sharpenratio': 0.3}
Starting multi-animal training....
```

</p></details>

**How to Reproduce the problem**
1. Go to train network
2. Click on 'OK'
3. Scroll down to 'Index error'

**Screenshots**

**Additional context**

-------

Thank you so much all the support-

-Ain

**Comment by jeylau** [comment]
Hey @deepbraincut (cool pseudo)! This bug is fixed with #1297, and will come together with rc3 (unless you pull the current code) 😊

**Comment by MMathisLab** [comment]
also it says what the error is : `ValueError: Crop dimensions are larger than image size` ; when you create the training set, be sure you adjust crop parameters...
